File: src/me/seeking/module/player/Sprint.java
package me.seeking.module.player;

import me.seeking.event.EventTarget;
import me.seeking.event.events.EventUpdate;
import me.seeking.module.Module;
import net.minecraft.client.settings.KeyBinding;
import org.lwjgl.input.Keyboard;

/**
 * This file is a part of Seeking Client.
 * @author MiLiBlue
 */
public class Sprint extends Module {
    public Sprint(){
        super("Sprint", ModuleType.Player);
    }

    @EventTarget
    public void onUpdate(EventUpdate e){
        KeyBinding.setKeyBindState(mc.gameSettings.keyBindSprint.getKeyCode(), true);
    }
}

File: src/main/java/me/seeking/module/player/Sprint.kt
package me.seeking.module.player

import me.seeking.event.EventTarget
import me.seeking.event.events.EventUpdate
import me.seeking.module.Module
import net.minecraft.client.settings.KeyBinding
import net.minecraft.network.play.client.C01PacketChatMessage

class Sprint:Module("Sprint", ModuleType.Player) {
    @EventTarget
    fun onUpdate(e:EventUpdate){
        if(mc.gameSettings.keyBindForward.isKeyDown)
            KeyBinding.setKeyBindState(mc.gameSettings.keyBindSprint.keyCode, true)
    }
}